src/legacy/core_plugins/interpreter/public/types/style.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FontLabel } from '../../common/lib/fonts';
+export { FontLabel as FontFamily, FontValue } from '../../common/lib/fonts';
+
+/**
+ * Type containing font weights.
+ */
+export enum FontWeight {
+  NORMAL = 'normal',
+  BOLD = 'bold',
+  BOLDER = 'bolder',
+  LIGHTER = 'lighter',
+  ONE = '100',
+  TWO = '200',
+  THREE = '300',
+  FOUR = '400',
+  FIVE = '500',
+  SIX = '600',
+  SEVEN = '700',
+  EIGHT = '800',
+  NINE = '900',
+}
+
+/**
+ * Type containing valid text alignments.
+ */
+export type TextAlignment = 'center' | 'left' | 'right' | 'justify';
+
+/**
+ * Collection of text alignments.
+ */
+export const TEXT_ALIGNMENTS: TextAlignment[] = ['center', 'left', 'right', 'justify'];
```

Comment:
This should be deleted and callsites converted to `TextAlignment.values()`... no need to repeat.  This was a chore I never cleaned up.

Reply:
is there a way to convert string literal union to js array ?

Reply:
> is there a way to convert string literal union to js array ?

Yeah IDK if you can do this. I think you can do it the other way though (define a js array and convert it to a string literal union). But I'm not aware of any way to go union > array.

I'm also not sure `TextAlignment.values()` would work in this case, but @clintandrewhall probably knows of some magic I haven't heard of here